.github/workflows/publish.yaml

```diff
@@ -25,13 +25,17 @@ jobs:
       - name: Get TAG
         id: get_tag
         run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}
+      - name: Set Username/Repo and ImagePrefix as ENV vars
+        run: |
+          echo "USER_REPO"=$(echo "$GITHUB_REPOSITORY" | awk '{print tolower($1)}' | sed -e "s/:refs//") >> $GITHUB_ENV && \
+          echo "IMAGE_PREFIX"=$(echo "ghcr.io/$GITHUB_REPOSITORY" | awk '{print tolower($1)}' | sed -e "s/:refs//") >> $GITHUB_ENV
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
       - name: Login to Docker Registry
         uses: docker/login-action@v1
         with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
+          username: ${{ github.repository_owner }}
```

Comment:
I have a feeling that this needs to be my username, like it is now because the token is pushing on behalf of myself..

Let's see if we can find a version of this that works this way already.

Reply:
My version just failed using openfaas as the username and secrets.GITHUB_TOKEN

https://github.com/openfaas/store-functions/runs/3360616559?check_suite_focus=true